cpp/include/cuspatial/experimental/ranges/multipoint_range.cuh

```diff
@@ -51,6 +51,8 @@ class multipoint_range {
   using point_t       = iterator_value_type<point_it_t>;
   using element_t     = iterator_vec_base_type<point_it_t>;
 
+  int32_t INVALID_IDX = -1;
```

Comment:
const or constexpr? 

```suggestion
  constexpr int32_t INVALID_IDX = -1;
```

Reply:
`constexpr`

Reply:
Wait, this wasn't meant to be added since it's not used. There is `INVALID_INDEX` in `multipolygon_range`.

Reply:
Removed